.gitlab/trigger_release/trigger_release.yml

```diff
@@ -18,7 +18,7 @@
   script:
     # agent-release-management creates pipeline for both Agent 6 and Agent 7
     # when triggered with major version 7
-    - RELEASE_VERSION="$(inv agent.version --major-version 7 --url-safe --omnibus-format)-1" || exit $?
+    - RELEASE_VERSION="$(inv agent.version --major-version 7 --url-safe --omnibus-format)-1" || exit $?; export RELEASE_VERSION
```

Comment:
Why not simply moving the export back to the beginning of the line? It would be more readable IMO and wouldn't change the behavior AFAICS

Reply:
If we do `EXPORT VARIABLE=$(command)` this will always succeed (because the `export` does) so we cannot throw to Gitlab the error code of the `command`, which is what we want. It will replay the job if we have an issue with Emissary, see #29262 description for more details

Reply:
Ah ok got it! Thanks for the explanation!